mdadm.h

```diff
@@ -2021,6 +2021,10 @@ enum r0layout {
 #define PATH_MAX	4096
 #endif
 
+/* Define length of char array which will contain int converted.
+ */
+#define INT_MAX_LEN 12
```

Comment:
The name is not right, it should be something like `INT_DEC_STR_MAX`

This is system dependent setting because `sizeof(int)` may not be 32. I would prefer:
```c
/* The max string length necessary for decimal conversion, cannot be longer than count of bits */
#define INT_2_DEC_STR_MAX sizeof(int) * 8
// Here comes other examples, just FYI
/* For binary conversion, in the worst we need one char more */
#define INT_2_BIN_STR_MAX INT_2_DEC_STR_MAX + 1
/* One hex sign represent one byte */
#define INT_2_HEX_STR_MAX sizeof(int) + 1
```
Using hex is more convenient but we have decimal based API..